.devcontainer/devcontainer.json

```diff
@@ -1,14 +1,17 @@
 // See https://aka.ms/vscode-remote/devcontainer.json for format details.
 {
   "name": "itwinjs-core-linux",
-  "dockerFile": "./Dockerfile",
-  "context": ".",
```

Comment:
what happened to context, workspaceMount and workspaceFolder?

Reply:
Don't think those are needed. The dev container is working fine without them set. zsh starts off at `/workspaces/itwinjs-core` correctly